docs/extend/index.md

```diff
@@ -97,7 +97,7 @@ started without error.
     ```bash
     $ docker volume ls
 
-    DRIVER              NAME
+    DRIVER              VOLUME NAME
     vieux/sshfs         sshvolume
```

Comment:
@vdemeester oh, yes; the change wasn't intentional, so if you can change it back, that'd be good (I _do_ like just "name", but it should be discussed first if we want to change)